docs/migrate/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Migrate to OpenSearch
+nav_order: 6
+has_children: true
+has_toc: true
+---
+
+# Migrate to OpenSearch
+
+The process of migrating from Elasticsearch OSS (including Open Distro for Elasticsearch) to OpenSearch varies depending on your current version of Elasticsearch OSS, install type, tolerance for downtime, and cost-sensitivity. Rather than concrete steps to cover every situation, we have general guidance for the process.
```

Comment:
I think this is about "upgrading" to OpenSearch, not migrating.

Reply:
I don't think so, actually. If we take the user goal as "I have an Elasticsearch OSS cluster and would like to run OpenSearch instead," then there are a few ways to get there. "Upgrading" to OpenSearch is just one of them. Node replacement is another, and snapshot restore is probably the safest path. All approaches ultimately get you to the same place.

And of course, we'll eventually have an "Upgrade OpenSearch" section, so it seems worthwhile to put some distance between the two and not have the top-level be "Upgrade to OpenSearch."

<img width="1082" alt="migrate" src="https://user-images.githubusercontent.com/9122297/121087336-233c2500-c799-11eb-9f73-286c513075f2.png">

Reply:
I think this is worth a discussion. I feel pretty strongly that we want to call this "upgrading" because there's continuity in OSS versions, and upgrading doesn't require downtime.

Maybe @elfisher wants to take this discussion on?

Reply:
"Upgrade migration" sounds strange ;)

Reply:
> "Upgrade migration" sounds strange ;)

No objection on that, and I'll change the title for that page. But the top-level is still about getting to OpenSearch, not necessarily upgrading to get there.